anndata/core/aligned_mapping.py

```diff
@@ -259,11 +278,11 @@ def __init__(
         self.dim_names = parent_mapping.dim_names[subset_idx]
 
 
-AxisArraysBase._view_class = AxisArraysView
-AxisArraysBase._actual_class = AxisArrays
+AxisArrays.View = _AxisArraysView
```

Comment:
I think these should be private. It's very possible these classes won't exist in a version or two, so I'd like to expose very little interface.

Reply:
Well, everything not documented is private, but sure, let’s make this even clearer with underscores.